### DISCUSSION TITLE
StyleClass `enterClass` definition

### DISCUSSION THREAD

**Discussion by dalenguyen** [primefaces/community#337] [PrimeNG] [answered]
I have a question on `enterClass` definition, in the docs, it says: 

`enterClass | string | null | Style class to add when item begins to get displayed.`

However, in reality, when clicking on the button, the `enterClass` is removed when the button is clicked. Is it incorrect?

Link: https://primeng.org/styleclass

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

It's correct but I guess there is a missing piece in documentation, `enterClass` refers to the class to be targeted and where the animation enters from. `enterActiveClass` refers to the class will be applied afterwards.

Thanks for reporting it!

**Reply by dalenguyen** [reply]
Thanks for you reply. It makes sense now 🙏
